src/context/yaml/index.ts

```diff
@@ -61,14 +78,14 @@ export default class {
 
     // Run initial schema check to ensure valid YAML
     const auth0 = new Auth0(this.mgmtClient, this.assets, toConfigFn(this.config));
-    await auth0.validate('validate');
+    await auth0.validate();
```

Comment:
I assume this is intentional (removing the argument)

Reply:
Correct, the compiler was complaining because the `validate` method doesn't accept any arguments.